codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithypython/PythonTestModels.java

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.amazon.polymorph.smithypython;
+
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import software.amazon.polymorph.TestModelTest;
+
+class PythonTestModels extends TestModelTest {
+
+    private static final Set<String> DISABLED_TESTS = new HashSet<>();
+
+    static {
+        DISABLED_TESTS.add("AggregateReferences");
+        DISABLED_TESTS.add("LanguageSpecificLogic");
+        DISABLED_TESTS.add("SimpleTypes/BigDecimal");
+        DISABLED_TESTS.add("SimpleTypes/BigInteger");
+        DISABLED_TESTS.add("SimpleTypes/SimpleByte");
+        DISABLED_TESTS.add("SimpleTypes/SimpleFloat");
+        DISABLED_TESTS.add("SimpleTypes/SimpleShort");
+        DISABLED_TESTS.add("SimpleTypes/SimpleTimestamp");
+        DISABLED_TESTS.add("aws-sdks/ddb-lite");
```

Comment:
FYI the -lite models should be very easy to turn on as well, since the projection is handled before hitting any language-specific logic.

Reply:
Took a look, I'll leave these unimplemented for now.
I think Python and Go will need extra work to figure out how to pass in the "dependency-module-names" and "library-name" flags.

Reply:
Fair enough, there's no real motivation to get those working now anyway - they are mainly useful during the development of new languages and you're already past that stage.